loki/expression/expr_visitors.py

```diff
@@ -138,6 +138,17 @@ def visit_TypeDef(self, o, **kwargs):
         """
         return self._return(o, ())
 
+    def visit_VariableDeclaration(self, o, **kwargs):
+        expressions = ()
+        for v in o.symbols:
+            if v.type.initial is not None:
+                retrieved = self.retrieve(v.type.initial)
+                if retrieved:
+                    expressions += as_tuple(retrieved)
```

Comment:
Since `as_tuple` returns an empty tuple even for `None`, the following should suffice:
```suggestion
                retrieved = as_tuple(self.retrieve(v.type.initial))
```